frontend/app/services/work-package-service.js

```diff
@@ -75,6 +76,47 @@ module.exports = function($http,
   }
 
   var WorkPackageService = {
+    initializeWorkPackage: function(projectIdentifier, initialData) {
+      var changes = _.clone(initialData);
+      if (initialData.type) {
+        changes._links = changes.links || {};
```

Comment:
For one I don't get the (in)difference between `_links` and `links`, as well as `_embedded` and`embedded`. I usually try and see which fails.

This gives the line above a gentle, fishy smell. But I can't judge if the smell is okay or not.